libs/types/src/fixed_point.rs

```diff
@@ -412,20 +418,38 @@ pub trait FixedPointNumberExtension: FixedPointNumber {
 	}
 
 	/// Checked self raised to pow; rounds precision to floor.
-	/// Saturates if result out of bounds.
+	/// SatuFixedU128s if result out of bounds.
 	fn saturating_pow_floor(self, pow: usize) -> Self {
 		self.saturating_pow_with_rounding(pow, SignedRounding::Minor)
 	}
 
 	/// Checked self raised to pow; rounds precision to ceil.
-	/// Saturates if result out of bounds.
+	/// SatuFixedU128s if result out of bounds.
 	fn saturating_pow_ceil(self, pow: usize) -> Self {
 		self.saturating_pow_with_rounding(pow, SignedRounding::Major)
 	}
 }
 
+const DECIMALS_18: u128 = 1_000_000_000_000_000_000u128;
+const DECIMALS_27: u128 = 1_000_000_000_000_000_000_000_000_000u128;
+
+/// A fixed point number representation with 27 decimals. Used for
+/// representing a rate, mostly interest rate.
+#[doc = "_Fixed Point 128 bits unsigned with 27 decimals precision"]
+pub type Rate = FixedU128<DECIMALS_27>;
+
+/// A fixed point number representation with 18 decimals. Used for
+/// representing a ratio between two things.
+#[doc = "_Fixed Point 128 bits unsigned with 18 decimals precision"]
+pub type Ratio = FixedU128<DECIMALS_18>;
+
+/// A fixed point number representation with 18 decimals. Used for
+/// representing a quantity of something.
+#[doc = "_Fixed Point 128 bits unsigned with 18 decimals precision"]
+pub type Quantity = FixedU128<DECIMALS_18>;
```

Comment:
Out of curiosity, where is this expected to be used?

Reply:
When you want to express an absolute number value with decimals. `Ratio` should represent instead a proportion of a value. Both are represented in the same way and are the same.

Maybe these "aliases" could be moved to our runtime definition, because they are how we are using them.

Reply:
I am not keen on moving them now as this means adapting all imports of `Rate`. Lets do in a follow up if wanted.